templates/Makefile

```diff
@@ -135,12 +170,34 @@ $(ENMASSE_WITH_STANDARD_AUTHSERVICE_BUNDLE): replace_images
 		mv $$i.tmp $$i; \
 	done
 
-install: ansible_install component_install $(ENMASSE_WITH_STANDARD_AUTHSERVICE_BUNDLE) $(ENMASSE_WITH_NONE_AUTHSERVICE_BUNDLE) $(ENMASSE_WITH_STANDARD_AUTHSERVICE_TEMPLATE) $(ENMASSE_WITH_NONE_AUTHSERVICE_TEMPLATE)
+ENMASSE_WITH_NONE_AUTHSERVICE_MONITORING_BUNDLE=$(PACKAGE_INSTALL_DIR)/bundles/enmasse-with-none-authservice-monitoring
+$(ENMASSE_WITH_NONE_AUTHSERVICE_MONITORING_BUNDLE): replace_images
```

Comment:
I would just create only the enmasse-with-standard-authservice-monitoring bundle and template, just drop the none-authservice version.